src/modules/listClass.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable linebreak-style */
+const list = JSON.parse(localStorage.getItem('listStorage')) || [];
+const addListItem = document.getElementById('input-task');
+const listContainer = document.querySelector('.todolist');
+
+export const trashCan = document.querySelectorAll('.delete');
+
+export default class Todolist {
+  constructor(description, complete, index) {
+    this.description = description;
+    this.complete = complete;
+    this.index = index;
+  }
+
+  static createList = () => {
+    const listContainer = document.querySelector('.todolist');
+    const displayList = list.map((list, index) => `<li class="list-item " id="${index}"><div class="inline"><input id="box-check" type=checkbox name="checkbox" class="check-box"></div> <input type="text" class="to-do input-text" value="${list.description}"><div class="ellipsis-container"><i class="fa-solid fa-ellipsis-vertical"></i><i class="fa-solid fa-trash-can icon delete"></i></div></li>`).join('');
+    listContainer.innerHTML = displayList;
+    const ellipsis = document.querySelectorAll('.fa-ellipsis-vertical');
+    const removeItem = document.querySelectorAll('.delete');
+    const checkedBox = document.querySelectorAll('.check-box');
+    removeItem.forEach((del, i) => {
+      del.addEventListener('click', () => {
+        Todolist.deleteItem(i);
+      });
+    });
+    for (let i = 0; i < ellipsis.length; i += 1) {
+      ellipsis[i].addEventListener('click', () => {
+        removeItem[i].style.display = 'inline';
+        ellipsis[i].style.display = 'none';
+      });
```

Comment:
I think using ES6 forEach() method instead of for loop would have been better because we are urged to use ES6 syntax as much as possible.